docs/src/pages/components/transfer-list/TransferList.js

```diff
@@ -1,23 +1,16 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
 import ListItemText from '@material-ui/core/ListItemText';
 import Checkbox from '@material-ui/core/Checkbox';
-import Button from '@material-ui/core/Button';
+import MuiButton from '@material-ui/core/Button';
 import Paper from '@material-ui/core/Paper';
 
-const useStyles = makeStyles((theme) => ({
-  paper: {
-    width: 200,
-    height: 230,
-    overflow: 'auto',
-  },
-  button: {
-    margin: theme.spacing(0.5, 0),
-  },
+const Button = styled(MuiButton)(({ theme }) => ({
```

Comment:
Did you consider using the sx prop directly? It seems that we could either use the stack pattern or the `my` key.

Reply:
I didn't want to do it so as not to repeat the code more than once. But based on the comments of the floating action button PR, I will modify that part to use the prop `sx`

Reply:
Resolved